components/app_bar/__snapshots__/app_bar.test.tsx.snap

```diff
@@ -159,3 +159,5 @@ exports[`components/app_bar/app_bar should match snapshot on mount 1`] = `
   </div>
 </AppBar>
 `;
+
+exports[`components/app_bar/app_bar should match snapshot on mount when App Bar is disabled 1`] = `<AppBar />`;
```

Comment:
Should the `AppBar` even be rendered when it's disabled? This case is probably fine since I don't think `getAppBarPluginComponents` or `getAppBarBindings` are particularly complex to run, but it's something I want to keep in mind for components that are always mounted but only visible sometimes.